frontend/public/components/build.tsx

```diff
@@ -121,6 +121,7 @@ export enum BuildStrategyType {
   JenkinsPipeline = 'JenkinsPipeline',
   Source = 'Source',
   Pac = 'Pac',
+  ServerlessFunction = 'ServerlessFunction',
```

Comment:
should not have this in public but rather extend it in dev-console/knative as needed